VideoLocker/src/main/java/org/edx/mobile/util/images/ErrorUtils.java

```diff
@@ -32,8 +34,14 @@ public static String getErrorMessage(@NonNull Throwable ex, @NonNull Context con
                 case HTTP: {
                     if (cause.getResponse() != null) {
                         final int status = cause.getResponse().getStatus();
-                        if (status == 503) {
-                            errorMessage = context.getString(R.string.network_service_unavailable);
+                        switch (status) {
```

Comment:
You can remove the variable, and replace the `switch` subject with the method call directly for conciseness, e.g. `switch (cause.getResponse().getStatus()`).